activation/nipost.go

```diff
@@ -283,7 +304,14 @@ func (nb *NIPostBuilder) BuildNIPost(
 		}
 	}
 
-	// Phase 2: Post execution.
+	// Stage 2: Post execution.
+	if err := nb.identityStates.Set(signer.NodeID(), types.IdentityStatePostProving); err != nil {
+		nb.logger.Warn("failed to switch identity state",
+			zap.Stringer("smesherID", signer.NodeID()),
+			zap.Error(err),
+		)
```

Comment:
Also, do we need to continue here?

Reply:
Yeap (imo) https://github.com/spacemeshos/go-spacemesh/pull/6266#discussion_r1723930653